components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,145 @@ toc = true
+<!--The TOC below is auto-generated by the VSCode "Markdown TOC" plugin; do not edit manually.
+With the plugin installed, this TOC will update automatically when you save the file! -->
+<!-- TOC depthFrom:2 depthTo:3 -->
+
+- [General Notes](#general-notes)
+- [Notes on the Projects Property](#notes-on-the-projects-property)
+- [Policies](#policies)
+  - [Listing Policies](#listing-policies)
+  - [Getting a Policy](#getting-a-policy)
+  - [Creating a Policy](#creating-a-policy)
+  - [Updating a Policy](#updating-a-policy)
+  - [Deleting a Policy](#deleting-a-policy)
+- [Policy Membership](#policy-membership)
+  - [Listing Policy Members](#listing-policy-members)
+  - [Updating All Members on a Policy](#updating-all-members-on-a-policy)
+  - [Adding Members to a Policy](#adding-members-to-a-policy)
+  - [Removing Members from a Policy](#removing-members-from-a-policy)
+- [Roles](#roles)
+  - [Listing Roles](#listing-roles)
+  - [Getting a Role](#getting-a-role)
+  - [Creating a Role](#creating-a-role)
+  - [Updating a Role](#updating-a-role)
+  - [Deleting a Role](#deleting-a-role)
+- [Projects](#projects)
+  - [Listing Projects](#listing-projects)
+  - [Getting a Project](#getting-a-project)
+  - [Creating a Project](#creating-a-project)
+  - [Updating a Project](#updating-a-project)
+  - [Deleting a Project](#deleting-a-project)
+- [Project Rules](#project-rules)
+  - [Listing Rules for a Project](#listing-rules-for-a-project)
+  - [Getting a Project Rule](#getting-a-project-rule)
+  - [Creating a Project Rule](#creating-a-project-rule)
+  - [Updating a Project Rule](#updating-a-project-rule)
+  - [Deleting a Project Rule](#deleting-a-project-rule)
+- [Users](#users)
+  - [Listing Users](#listing-users)
+  - [Getting a User](#getting-a-user)
+  - [Creating a User](#creating-a-user)
+  - [Updating a User](#updating-a-user)
+  - [Deleting a User](#deleting-a-user)
+  - [List Teams for a User](#list-teams-for-a-user)
+  - [Updating Your Own User](#updating-your-own-user)
+- [Teams](#teams)
+  - [Listing Teams](#listing-teams)
+  - [Getting a Team](#getting-a-team)
+  - [Creating a Team](#creating-a-team)
+  - [Updating a Team](#updating-a-team)
+  - [Deleting a Team](#deleting-a-team)
+- [Team Membership](#team-membership)
+  - [Listing Team Users](#listing-team-users)
+  - [Adding Users to a Team](#adding-users-to-a-team)
+  - [Removing Users from a Team](#removing-users-from-a-team)
+- [Tokens](#tokens)
+  - [Listing Tokens](#listing-tokens)
+  - [Getting a Token](#getting-a-token)
+  - [Creating a Token](#creating-a-token)
+  - [Updating a Token](#updating-a-token)
+  - [Deleting a Token](#deleting-a-token)
+- [Restoring Admin Access](#restoring-admin-access)
+
+<!-- /TOC -->
+
+## General Notes
+
+1. If you do not have one already, generate an admin token for IAM v2.
+   (Neither The v1 command `chef-automate admin-token` nor any v1 admin tokens will work for v2.)
```

Comment:
Is this true? I think those v1 tokens _do_ work...

Reply:
I had confirmed it; it is true.

Reply:
I just tested locally and I'm still seeing the v1 token work fine (the reverse is not true-- a v2 token will not work when you reset), but all v1 admin tokens are added to the admin team (or policy) on upgrade.

Mind double checking on your end?

Reply:
Hmm, OK, let's go with "they work" then. I will go back and check my machine when I have a chance.